### PR TITLE
issue: gtest doca WIP

### PR DESCRIPTION
**Pull request by tomerdbz**
## Description
Please provide a summary of the change.

##### What
_Subject: what this PR is doing in one line._ 

##### Why ?
_Justification for the PR. If there is existing issue/bug please reference._

##### How ?
_It is optional but for complex PRs please provide information about the design,
architecture, approach, etc._

## Change type
What kind of change does this PR introduce?
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Tests
- [ ] Other

## Check list
- [ ] Code follows the style de facto guidelines of this project
- [ ] Comments have been inserted in hard to understand places
- [ ] Documentation has been updated (if necessary)
- [ ] Test has been added (if possible)

### CODE REVIEW

**Comment by dpressle** [comment]
You know you should enable gtest in CI if you want to run it? Uncomment these blocks:
https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L110
https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L314

File: contrib/jenkins_tests/gtest.sh
Comment (dpressle) [inline contrib/jenkins_tests/gtest.sh:1]: IPv6 is not supported here please comment all the lines with ipv6

**Review by dpressle — CHANGES_REQUESTED** [review]

**Comment by tomerdbz** [comment]
> You know you should enable gtest in CI if you want to run it? Uncomment these blocks: https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L110 https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L314

@dpressle still not running. what am i missing?

**Comment by dpressle** [comment]
> > You know you should enable gtest in CI if you want to run it? Uncomment these blocks: https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L110 https://github.com/Mellanox/libxlio/blob/doca_xlio_vNext/.ci/matrix_job.yaml#L314
> 
> @dpressle still not running. what am i missing?

you were supposed to uncomment not to change it, we dont run on beni09 we run on container.

**Pull request closed without merging**